endaq/plot/dashboards.py

```diff
@@ -128,19 +128,6 @@ def rolling_enveloped_dashboard(
     for channel_data in channel_df_dict.values():
         window = int(np.around((channel_data.shape[0]-1) / desired_num_points, decimals=0))
 
-        # If a window size of 1 is determined, it sets the stride to 1 so we don't get an error as a result of the
-        # 0 length stride
-        stride = 1 if window == 1 else window - 1
-
-        rolling_n = channel_data.rolling(window)
-        min_max_tuple = (rolling_n.min()[::stride], rolling_n.max()[::stride])
-        min_max_equal = min_max_tuple[0] == min_max_tuple[1]
-
-        is_nan_min_max_mask = np.logical_not(
-            np.logical_and(
-                pd.isnull(min_max_tuple[0]),
-                pd.isnull(min_max_tuple[1])))
-
```

Comment:
moving this chunk has no functional effect, right? It's just for organization?

Reply:
or I guess the use of `channel_data` might rely on the new index that was set?

Reply:
The reason it matters is that the `rolling` method of the channel DataFrame was being used _before_ the index was set properly.   The other issue that was fixed is that there was a line calling `set_index` without setting `inplace=True`.